Makefile

```diff
@@ -20,6 +20,7 @@ endif # TARGET
 
 SERVICE := $(or ${SERVICE},quay.io/ocpmetal/assisted-service:latest)
 ISO_CREATION := $(or ${ISO_CREATION},quay.io/ocpmetal/assisted-iso-create:latest)
+DUMMY_IGNITION := $(or ${DUMMY_IGNITION},quay.io/ocpmetal/ignition-dummy:latest)
```

Comment:
We should never push the dummy ignition to ocpmetal, so the default should be something local. for example `minikube-local-registry/ignition-dummy-generator`

Reply:
fixing